### DISCUSSION TITLE
BZip2/xz support for input files

### DISCUSSION THREAD

**Discussion by ipetrushin** [ablab/spades#738] [Ideas]
Dear SPAdes team!

Due to large FASTQ files may I ask for bzip2/xz compression support for input files? 
.bz2 files up to 20-25% smaller than .gz, disk space on cluster is valuable and large data sets may not fit quotas.

Related benchmark paper [https://www.biorxiv.org/content/10.1101/642553v2.full](url)

Sincerely,
Ivan

**Comment by asl** [comment]
We may consider adding this is an option, yes, if the corresponding libraries are available on the system. Patches are always welcome :)
